Provide old #define for code not yet converted to the new ones set in
xkb-defaults.patch (like the xorgcfg & xorgconfig code that isn't in 1.6).

diff -urp -x '*~' -x '*.orig' include/xkb-config.h.in include/xkb-config.h.in
--- include/xkb-config.h.in	2009-01-22 20:32:43.368859000 -0800
+++ include/xkb-config.h.in	2009-01-22 20:32:51.502769000 -0800
@@ -7,6 +7,7 @@
 
 /* Default set of XKB rules. */
 #undef XKB_DFLT_RULES
+#define __XKBDEFRULES__ XKB_DFLT_RULES
 
 /* Default XKB model. */
 #undef XKB_DFLT_MODEL
diff -urp -x '*~' -x '*.orig' include/xorg-server.h.in include/xorg-server.h.in
--- include/xorg-server.h.in	2009-01-22 20:32:43.369262000 -0800
+++ include/xorg-server.h.in	2009-01-22 20:32:51.503745000 -0800
@@ -108,6 +108,7 @@
 
 /* XKB default rules */
 #undef XKB_DFLT_RULES
+#define __XKBDEFRULES__ XKB_DFLT_RULES
 
 /* Build XKB support in server */
 #undef XKB_IN_SERVER
